Makefile

```
@@ -42,6 +42,7 @@ install:											## Install the project and
 	if [ "$(VENV_EXISTS)" ]; then $(MAKE) clean; fi
 	@if [ "$(USING_PDM)" ]; then $(PDM) config venv.in_project true && python3 -m venv --copies .venv && . $(ENV_PREFIX)/activate && $(ENV_PREFIX)/pip install --quiet -U wheel setuptools cython pip; fi
 	@if [ "$(USING_PDM)" ]; then $(PDM) install -G:all; fi
+	@.venv/bin/pre-commit install
```

```suggestion
	@$(ENV_PREFIX)/pre-commit install
```
This should be where ENV_PREFIX is already pointed to.

---

If the virtual environment is being installed for the first time, then the `.venv` directory would not exit and so `ENV_PREFIX` would be an empty string.